extension/luaDebugSession.ts

```diff
@@ -262,28 +258,22 @@ export class LuaDebugSession extends LoggingDebugSession {
         this.updateLuaPath("LUA_PATH", processOptions.env, true);
 
         //Launch process
-        let processExecutable: string;
-        let processArgs: string[];
-        if (isCustomProgramConfig(this.config.program)) {
-            processExecutable = `"${this.config.program.command}"`;
-            processArgs = typeof this.config.args !== "undefined" ? this.config.args : [];
+        const processExecutable = `"${this.config.program.tarantool}"`;
+        const configArgs = (typeof this.config.args !== "undefined")
+            ? `, ${this.config.args.map(a => `[[${a}]]`)}`
+            : "";
+        const processArgs: string[] = [
+            "-e",
+            "\"require 'strict'.off()\"",
```

Comment:
Soon after our discussion I've realized that we do not need to mess with `strict` mode at all (even under debugging), as we do not use any global variable anymore (as it used to be before), but rather simple `require "tarantool".debug` which is perfectly compatible with `strict` mode.

i.e. those 2 lines adding call to "require 'strict'.off()" are not needed.

Reply:
Looks like we do need to have it enabled as there is `rawlen` used